DISCUSSION.md

```diff
@@ -116,7 +116,7 @@ We depend on access-atomicity in some cases, but this should not impact implemen
 
 TC39 discussed the memory model at several meetings and some desiderata were enumerated.  Here is a list, with some discussion.
 
-#### Correcness
+#### Correctness
```

Comment:
lol - chuckled a bit when I saw the typo, our brains should come with SAT solvers :sweat: